Network/HTTP/Req.hs

```diff
@@ -861,6 +871,19 @@ instance HttpMethod method => RequestComponent (Womb "method" method) where
 data Url (scheme :: Scheme) = Url Scheme (NonEmpty Text)
   -- NOTE The second value is the path segments in reversed order.
   deriving (Eq, Ord, Show, Data, Typeable, Generic)
+type role Url nominal
```

Comment:
I see you first added the role in a separate commit. What was the motivation for that?

Reply:
I did it in a separate commit, because it isn't really related to the other changes. Ideally it should be a separate PR, I just thought it's so insignificant and uncontroversial I'll just add it to this PR.

Also this actually removed again in my other PR, as that will cause the role to be correctly inferred as nominal.

Reply:
So perhaps we should just remove it here? The smaller the diff, the easier I'll merge the PR.

Reply:
Ok, I'll remove it.